### PR TITLE
New version: eza-community.eza version 0.20.14

### PR DESCRIPTION
**Pull request by cafkbot-winget**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/202261)

### CODE REVIEW

**Comment by stephengillie** [comment]
URL: https://github.com/eza-community/eza/releases/download/v0.20.14/eza.exe_x86_64-pc-windows-gnu.zip 
Status Code: 200

(Automated message - build 931)

File: manifests/e/eza-community/eza/0.20.14/eza-community.eza.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+PackageIdentifier: eza-community.eza
+PackageVersion: 0.20.14
+PackageLocale: en-US
+Publisher: eza community
+PublisherUrl: https://eza.rocks
+PublisherSupportUrl: https://github.com/eza-community/eza/issues
+PackageName: eza
+PackageUrl: https://github.com/eza-community/eza
+License: MIT
+LicenseUrl: https://github.com/eza-community/eza/blob/main/LICENCE
```
Comment (DuckDuckStudio) [inline manifests/e/eza-community/eza/0.20.14/eza-community.eza.locale.en-US.yaml:13]:
```suggestion
LicenseUrl: https://github.com/eza-community/eza/blob/main/LICENSE.txt
```

**Review by DuckDuckStudio — CHANGES_REQUESTED** [review]

**Comment by cafkafk** [comment]
maintainers should feel free to apply this change on behalf of the bot, e.g. ammend it or whatever other action deemed appropriate

File: manifests/e/eza-community/eza/0.20.14/eza-community.eza.locale.en-US.yaml (hunk 1 above)
Comment (DuckDuckStudio) [reply manifests/e/eza-community/eza/0.20.14/eza-community.eza.locale.en-US.yaml:13]: I don't have permission to write to bot's fork repository...

Comment (DuckDuckStudio) [reply manifests/e/eza-community/eza/0.20.14/eza-community.eza.locale.en-US.yaml:13]: And License should change to `European Union Public License 1.2`

File: manifests/e/eza-community/eza/0.20.14/eza-community.eza.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+PackageIdentifier: eza-community.eza
+PackageVersion: 0.20.14
+PackageLocale: en-US
+Publisher: eza community
+PublisherUrl: https://eza.rocks
+PublisherSupportUrl: https://github.com/eza-community/eza/issues
+PackageName: eza
+PackageUrl: https://github.com/eza-community/eza
+License: MIT
```
Comment (DuckDuckStudio) [inline manifests/e/eza-community/eza/0.20.14/eza-community.eza.locale.en-US.yaml:12]:
```suggestion
License: European Union Public License 1.2
```

Should the license also be modified?

**Review by DuckDuckStudio — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**